catalog/task/get-ocp-version/0.1/tests/test-get-ocp-version.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: test-get-ocp-version
+spec:
+  description: |
+    Run the get-ocp-version task with a fbcFragment FBC Image
+    and verify that OCP-Version is been stored as expected
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: run-task
+      taskRef:
+        name: get-ocp-version
+      params:
+        - name: fbcFragment
+          value: quay.io/hacbs-release-tests/managed-release-team-tenant/sample-fbc-application/sample-fbc-component@sha256:3bab7aa91d6e62c6e6f8cd6e701b78124d477eab3d8d1e1d3bce59c81204b1d7
+    - name: check-result
+      params:
+        - name: stored-version
+          value: $(tasks.run-task.results.stored-version)
+      runAfter:
+        - run-task
+      taskSpec:
+        params:
+          - name: stored-version
+        steps:
+          - name: check-result
+            image: quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31
+            env:
+              - name: "OCP_VERSION"
+                value: '$(params.stored-version)'
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              echo Task result contains the valid OCP version number
+              test $(echo $OCP_VERSION) == "v4.12"
```

Comment:
IIRC, we should here match if the given fromIndex in the releaseStrategy matches with the one in the built snapshot. if they not match, then you fail the task.